src/js/components/stock-movement/outbound/StockMovementOutboundList.jsx

```diff
@@ -21,6 +22,7 @@ const StockMovementOutboundList = (props) => {
   } = useOutboundFilters(props.isRequestsList);
 
   useTranslation('stockMovement', 'StockMovementType', 'reactTable');
+  useDeleteNotificationListener();
```

Comment:
I feel like now this hook is generic enough that it can be placed somewhere near the root element like for example on `Router.jsx` where we have the `useConnectivityListener`.
That way this feature would be available across all the react pages